### ISSUE TITLE
Import failing to import all entities

### ISSUE THREAD

**Issue by at055612** [gchq/stroom#442]
When importing all items in a zip that was the result of a full export (v5 => v5), a few problems are happening:
1. Not all entities are being imported
1. Seeing errors in the log (see below)
1. Some folders are being duplicated

```
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '13-Statistics Test Data' for key 'NAME'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
```

**Comment by at055612** [comment]
![screenshot from 2017-10-11 12-17-01](https://user-images.githubusercontent.com/22818309/31438059-71aeb1b2-ae7f-11e7-8719-fa6ad436c8fa.png)

**Comment by at055612** [comment]
![screenshot from 2017-10-11 12-20-14](https://user-images.githubusercontent.com/22818309/31438070-7e185bf6-ae7f-11e7-8ca2-2bea5412a9ed.png)

**Comment by at055612** [comment]
After re-importing the missing Apache content I now get
![image](https://user-images.githubusercontent.com/22818309/31438473-20ef4948-ae80-11e7-8528-f73467fcfa8b.png)

**Comment by at055612** [comment]
Can re-create this by 
1. running setupSampleData
1. Starting Stroom and exporting all entities
1. Trash the databases and start stroom without running setupSampleData
1. Import the file exported in the step above

Some folders are duplicated inside the folder that they duplicate.

**Comment by at055612** [comment]
The issues seems to be caused by some bad error handling and the presence of the the 'Old Internal Statistics' folder.  A fresh install of stroom will come with an 'Old Internal Statistics' folder (with a randomly generated uuid).  If the import includes that same folder, but with a different uuid it causes errors in the import.

**Comment by at055612** [comment]
Fixed in 5.0
